pkg/charts/package.go

```diff
@@ -86,6 +88,10 @@ func (p *Package) GeneratePatch() error {
 
 // GenerateCharts creates Helm chart archives for each chart after preparing it
 func (p *Package) GenerateCharts() error {
+	if p.DoNotRelease {
+		logrus.Infof("Skipping package marked doNotRelease")
+		return nil
+	}
```

Comment:
If kiali is the use case here, doesn't istio literally use the kiali tarball ? What I would think accomplishes that would be removing it from index.yaml. 
Maybe there is a better use case where the chart does get packaged in or isn't being used.

Reply:
The other use case is cilium @ rancher/rke2-charts#46. We want upstream cilium as a package subchart and patched and not to be consumed directly.

Reply:
> If kiali is the use case here, doesn't istio literally use the kiali tarball ? 

@cbron at the moment, no. 

Istio [re-declares Kiali](https://github.com/rancher/charts/tree/release-v2.5/charts/rancher-istio/rancher-istio/1.8.300/charts/kiali) within it [as a local chart](https://github.com/rancher/charts/blob/aae93d0125fad561024b25ac1a15580ea612e158/charts/rancher-istio/rancher-istio/1.8.300/requirements.yaml#L4), so they are completely independent of each other.

So we could safely remove it from going through the charts framework at all, unless there is a use case for installing Kiali alone.

Reply:
Interesting. No Kiali is strictly for Istio and it is hidden anyway, so we should probably do this.